python/servo/package_commands.py

```diff
@@ -287,24 +287,21 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
             msi_path = path.join(dir_to_msi, "Servo.msi")
             print("Packaged Servo into {}".format(msi_path))
         else:
-            dir_to_package = '/'.join(binary_path.split('/')[:-1])
-            dir_to_root = '/'.join(binary_path.split('/')[:-3])
-            resources_dir = dir_to_package + '/resources'
-            if os.path.exists(resources_dir):
-                delete(resources_dir)
-            shutil.copytree(dir_to_root + '/resources', resources_dir)
+            dir_to_temp = path.join(os.path.dirname(binary_path), 'targz')
```

Comment:
Also, instead of `targz`, let's use a more descriptive name like `packaging-temp`. This is also packaging-technology neutral, which will help us possibly combine these blocks down the road.